ffi/diplomat/src/timezone_formatter.rs

```diff
@@ -96,7 +98,120 @@ pub mod ffi {
             .into()
         }
 
+        #[diplomat::rust_link(
+            icu::datetime::TimeZoneFormatter::load_generic_non_location_long,
+            FnInStruct
+        )]
+        pub fn load_generic_non_location_long(
```

Comment:
nit: probably worth having a line of docs for these

Reply:
will fix in my next PR